packages/devtools_app/lib/src/screens/memory/panes/diff/widgets/classes_table_diff.dart

```diff
@@ -128,6 +126,9 @@ class _InstanceColumn extends ColumnData<DiffClassStats>
     }
   }
 
+  static late HeapDataObtainer before;
+  static late HeapDataObtainer after;
```

Comment:
instead of storing data objects on the ColumnData classes, store them in a controller that the column methods can look the data up from. We should avoid storing data UI components to avoid state management issues. Same for all columns below